portal_partner_select_all/models/event.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    exclude_public_holidays = fields.Boolean(
```

Comment:
I think this file belongs to another addon... :roll_eyes:

Reply:
:man_facepalming: